notebooks/photutils/03_aperture_photometry/03_aperture_photometry.ipynb

```diff
@@ -0,0 +1,970 @@
+    "\n",
+    "plt.yscale('log')\n",
+    "plt.xscale('log')\n",
+    "plt.title('Histogram of Source Photometry')\n",
+    "plt.xlabel(r'Flux Count Rate ({})'.format(xdf_image.unit.to_string('latex')))\n",
+    "plt.ylabel('Number of Sources')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(8, 5))\n",
+    "\n",
+    "plt.scatter(table['area'], values, alpha=0.5)\n",
+    "\n",
+    "plt.yscale('log')\n",
+    "plt.xscale('log')\n",
+    "plt.title('Count Rate v. Aperture Area')\n",
+    "plt.xlabel('Aperture Area [pixels$^2$]')\n",
+    "plt.ylabel(r'Flux Count Rate ({})'.format(xdf_image.unit.to_string('latex')))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "\n",
+    "<h3>Exercise:</h3><br>\n",
+    "\n",
+    "Re-calculate the photometry for these elliptical apertures - or just a subset of them - using the <code>subpixel</code> aperture placement method instead of the default <code>exact</code> method. How does this affect the count sum calculated for those apertures?\n",
+    "\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "# Aperture Corrections"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We've done photometry with some lovely apertures, but unfortunately even using elliptical apertures with unique sizes and orientations does not account for an important source of extraneous flux: the sky background.\n",
+    "\n",
+    "## Local Background Subtraction\n",
+    "\n",
+    "In the [background estimation notebook](../01_background_estimation/01_background_estimation.ipynb), we explored how to perform global background subtraction of image data with `photutils`. However, you can also use `photutils` to perform local background estimations for aperture corrections.\n",
+    "\n",
+    "To estimate the local background for each aperture, measure the counts within annulus apertures around (but not including!) each source. In our example, we defined elliptical apertures with `r = 3` pixels to measure the counts within each source. To calculate the background for each source, let's measure the counts elliptical annuli between `r = 3.5` pixels and `r = 5` pixels."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "r_in = 3.5  # approximate isophotal extent of inner semimajor axis\n",
+    "r_out = 5.  # approximate isophotal extent of inner semimajor axis\n",
+    "\n",
+    "# Create the apertures\n",
+    "elliptical_annuli = []\n",
+    "for obj in catalog:\n",
+    "    position = (obj.xcentroid.value, obj.ycentroid.value)\n",
+    "    a_in = obj.semimajor_axis_sigma.value * r_in\n",
```

Comment:
The text says we're doing background subtraction between 3.5 and 5 (or 6 and 10) pixels, but these lines imply we're doing the background subtraction from 3.5 and 5 _semimajor axes_. This results in some very large ellipses annulus widths for the big galaxies, and widths which vary with the size of the object; should these not be something like `a_in = obj.semimajor_axis_sigma.value * large_number + r_in` with large_number ~ 5? You then get the difference between `r_out` and `r_in` with `obj.semimajor_axis_sigma.value * large_number` cancelling, giving a fixed number pixel width for all apertures as stated in the text.